tests/integration/test_case.go

```diff
@@ -220,6 +227,22 @@ type CreateView struct {
 	ExpectedError string
 }
 
+type RefreshViews struct {
```

Comment:
suggestion: would be nice to have the struct documented

Reply:
sorted